src/components/Toggle/Toggle.module.css

```diff
@@ -3,14 +3,17 @@
   height: 1.25rem;
   border-radius: 3rem;
   border-radius: 1.5rem;
-  padding: 2px;
+  padding: 0.125rem;
```

Comment:
I don't think the borders need to be expanded or contracted according to font size, but what do you think?